addons/links/src/preview.test.ts

```diff
@@ -1,9 +1,11 @@
 import addons from '@storybook/addons';
 import { SELECT_STORY } from '@storybook/core-events';
 
-import { __STORYBOOK_STORY_STORE__ } from 'global';
+import globalPkg from 'global';
```

Comment:
I had to name this one slightly differently due to jest wanting to mock the actual `global` below.